compiler-rt/lib/sanitizer_common/tests/sanitizer_block_signals.cpp

```diff
@@ -0,0 +1,81 @@
+//===-- sanitizer_block_signals.cpp ---------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file is a part of sanitizer_common unit tests.
+//
+//===----------------------------------------------------------------------===//
+#include <signal.h>
+#include <stdio.h>
+
+#include "gtest/gtest.h"
+#include "sanitizer_common/sanitizer_linux.h"
+
+namespace __sanitizer {
+
+#if SANITIZER_LINUX
+volatile int received_sig = -1;
+
+void signal_handler(int signum) { received_sig = signum; }
+
+TEST(SanitizerCommon, BlockSignals) {
+  // No signals blocked
```

Comment:
Instead of large test with 3 `{}`
please split into 3
```
TEST(SanitizerCommon, BlockSignal{Name1, Name2, Name3}) {
}
````

Reply:
Done